calcom/cal.com
# feat: signup refactor

Redesigns settings 

FIGMA: https://www.figma.com/file/UajFu4M1APhn2AywAEJkJr/New-Features?type=design&node-id=5237-479368&mode=design

How to test: 
Testing as a self hoster 
- run yarn dev 
- visit localhost:3000/signup
- Signup with premium username
- signup with normal username
- signup with team invite

Testing as a calcom hoster
- yarn dev
- run website - yarn dev in apps/website
- Signup with premium username -> See checkout flow
- signup with normal username
- signup with team invite

- ensure signup with google works correctly
- ensure sign up with saml works correctly
- ensure query params are filled from cal.com landing page button works correctly
- ensure email verification is sent 





Self hosted:
![CleanShot 2023-09-20 at 10 57 41](https://github.com/calcom/cal.com/assets/55134778/bf2b8b42-9f2b-44ec-9cdc-ccdee399a07f)



Screenshots

Ipad pro 11
![Sizzy-iPad Pro 11 localhost 21Sep 11 27](https://github.com/calcom/cal.com/assets/55134778/f2424b79-adbe-4726-82e6-3f03e771540e)
Iphone 14
![Sizzy-iPhone 14 localhost 21Sep 11 27](https://github.com/calcom/cal.com/assets/55134778/bf162f83-2742-42e3-9c6d-1ba00a182098)
Iphone 8
![Sizzy-iPhone 8 localhost 21Sep 11 27](https://github.com/calcom/cal.com/assets/55134778/cb76e476-7b66-4e80-81fc-23db346e85de)
Macbook Air
![Sizzy-MacBook Air localhost 21Sep 11 27](https://github.com/calcom/cal.com/assets/55134778/efd26630-c7f7-41f6-8e25-8ec98111bd78)
Ultrawide
![CleanShot 2023-09-21 at 11 28 05](https://github.com/calcom/cal.com/assets/55134778/c0d3ae61-e720-41a8-9e83-de74d93f534d)
